docs/apis/cli.md

```diff
@@ -187,6 +187,8 @@ Action "run" compiles and runs a program.
                                           java.net.URLClassLoader}.
      -d,--detached                        If present, runs the job in detached
                                           mode
+     -D,--configDir </path/to/confdir>    The configuration directory with which
```

Comment:
I totally agree, I've added a short option 'cause I think it was required. I will remove asap